README.md

```diff
@@ -215,84 +215,221 @@ result = mq.index("my-first-index").search('adventure', searchable_attributes=['
 
 ```python
 
-results = mq.index("my-first-index").delete_documents(ids=["article_591", "article_602"])
+settings = {
+    "treat_urls_and_pointers_as_images":True,   # allows us to find an image file and index it 
+    "model":"ViT-L/14"
+}
+response = mq.create_index("my-multimodal-index", **settings)
 
 ```
 
-### Delete index
-
-Delete an index.
+Images can then be added within documents as follows. You can use urls from the internet (for example S3) or from the disk of the machine:
 
 ```python
 
-results = mq.index("my-first-index").delete()
+response = mq.index("my-multimodal-index").add_documents([{
+    "My Image": "https://upload.wikimedia.org/wikipedia/commons/thumb/b/b3/Hipop%C3%B3tamo_%28Hippopotamus_amphibius%29%2C_parque_nacional_de_Chobe%2C_Botsuana%2C_2018-07-28%2C_DD_82.jpg/640px-Hipop%C3%B3tamo_%28Hippopotamus_amphibius%29%2C_parque_nacional_de_Chobe%2C_Botsuana%2C_2018-07-28%2C_DD_82.jpg",
+    "Description": "The hippopotamus, also called the common hippopotamus or river hippopotamus, is a large semiaquatic mammal native to sub-Saharan Africa",
+    "_id": "hippo-facts"
+}])
 
 ```
 
-## Multi modal and cross modal search
+You can then search using text as usual. Both text and image fields will be searched:
 
-To power image and text search, Marqo allows users to plug and play with CLIP models from HuggingFace. **Note that if you do not configure multi modal search, image urls will be treated as strings.** To start indexing and searching with images, first create an index with a CLIP configuration, as below:
+```python
+
+results = mq.index("my-multimodal-index").search('animal')
+
+```
+ Setting `searchable_attributes` to the image field `['My Image'] ` ensures only images are searched in this index:
 
 ```python
 
-settings = {
-  "treat_urls_and_pointers_as_images":True,   # allows us to find an image file and index it 
-  "model":"ViT-L/14"
-}
-response = mq.create_index("my-multimodal-index", **settings)
+results = mq.index("my-multimodal-index").search('animal',  searchable_attributes=['My Image'])
 
 ```
 
-Images can then be added within documents as follows. You can use urls from the internet (for example S3) or from the disk of the machine:
+### Searching using an image
+Searching using an image can be achieved by providing the image link.
 
 ```python
 
-response = mq.index("my-multimodal-index").add_documents([{
-    "My Image": "https://upload.wikimedia.org/wikipedia/commons/thumb/f/f2/Portrait_Hippopotamus_in_the_water.jpg/440px-Portrait_Hippopotamus_in_the_water.jpg",
-    "Description": "The hippopotamus, also called the common hippopotamus or river hippopotamus, is a large semiaquatic mammal native to sub-Saharan Africa",
-    "_id": "hippo-facts"
-}])
+results = mq.index("my-multimodal-index").search('https://upload.wikimedia.org/wikipedia/commons/thumb/9/96/Standing_Hippopotamus_MET_DP248993.jpg/1920px-Standing_Hippopotamus_MET_DP248993.jpg')
```

Comment:
same comment as above regarding wikipedia image stability

Reply:
Same as my reply to your other comment